modules/launcher/src/main/java/org/mule/module/launcher/artifact/DefaultResourceReleaser.java

```diff
@@ -34,6 +40,11 @@ private void deregisterJdbcDrivers()
             {
                 logger.debug("Deregistering driver: {}", driver.getClass());
                 DriverManager.deregisterDriver(driver);
+
+                if (isOracleDriver(driver))
+                {
+                    deregisterOracleDiagnosabilityMBean();
```

Comment:
I don't think you can have 2 oracle drivers in the same app. If that happens, there would be only a mbean and it would be registered, second attempt will fail silently, but because this is on application disposal there is no harm. But again, I don't think this is a vlid scenario